src/activities/guesscount-summer/GuesscountSummer.qml

```diff
@@ -0,0 +1,351 @@
+            property bool solved
+            property bool levelchanged : false
+            property var levelArr
+            property alias load: dialogActivityConfig
+            property alias operators: items.levelArr
+            property string mode
+            property int currentlevel
+        }
+
+        onStart:  if (activity.needRestart) {
+                      Activity.start(items);
+                      activity.needRestart = false;
+                  } else
+                      Activity.initLevel();
+        onStop: { Activity.stop() }
+
+
+        JsonParser {
+            id: parser
+            onError: console.error("Guesscount: Error parsing JSON: " + msg);
+        }
+
+        Loader {
+            id: admin
+            active: false
+            sourceComponent: Column {
+                spacing: 10
+                width: parent.width
+                height: parent.height
+
+                Repeater{
+                    id:levels
+                    model: Activity.numberOfLevel
+                    Admin{
+                        id:level
+                        level: modelData+1
+                        width: parent.width
+                        height: parent.height
+                        data: items.data
+                    }
+                }
+            }
+        }
+
+        Rectangle {
+            id: row1
+            width: parent.width
+            height: parent.height/10
+            anchors{
+                top: parent.top
+                topMargin: (parent.height/80)*3
+            }
+            color: "transparent"
+            Rectangle {
+                id: questionNo
+                width: parent.width*0.328;
+                height: parent.height;
+                radius: 20.0;
+                color: "steelblue"
+                anchors{
+                    left: parent.left
+                    leftMargin: parent.width*0.028
+                }
+                GCText{
+                    anchors.horizontalCenter: parent.horizontalCenter
+                    anchors.verticalCenter: parent.verticalCenter
+                    fontSize: mediumSize
+                    text: qsTr("%1/%2").arg(items.sublevel).arg(items.data.length)
+                }
+            }
+            Rectangle{
+                width: parent.width*0.35;
+                height: parent.height;
+                radius: 20
+                color: "orange"
+                anchors{
+                    right: parent.right
+                    rightMargin: parent.width*0.028
+                }
+                GCText{
+                    id: guess
+                    anchors.horizontalCenter: parent.horizontalCenter
+                    anchors.verticalCenter: parent.verticalCenter
+                    fontSize: mediumSize
+                    text: qsTr("Guesscount: %1").arg(items.result)
+                }
+            }
+        }
+
+        Column {
+            spacing: 10
```

Comment:
if you reduce the screen at minimum and have all the operands (let's say that for levle one, you selected admin mode and all the operands), we don't see the last numbers.